utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java

```diff
@@ -30,10 +30,10 @@
 @Slf4j
 public class DbValidationSteps {
 
-    private final DbUtils dbUtils;
+    private DbUtils dbUtils;
 
     public DbValidationSteps() {
-        dbUtils = new DbUtils(SampleDbConnectionManager.getConnection());
```

Comment:
Can we please maintain current idea about SampleDbConnectionManager having static method getConnection()?
The whole point was to not create db connection with every request/step and it was very well readable.
Your changes broke the pattern. DbUtils now need parameter to differentiate between mysql and postgre. What about leaving DbUtils as it is with connection parameter and change
``SampleDbConnectionManager.getConnection()``
to
``SampleDbConnectionManager.getConnection("databaseType")``

and implement whole logic inside of getConnection? I find your implementation chaotic.
One switch based on "databaseType" and one if-then-else per type would be enough.

With this approach you can also set it up that getConnection without data type will be default postgre to stay backward compatible.

Reply:
just fyi, I don't create new DB connection on each request/step, but I generally, let's say your proposal is more clear...

Reply:
done